election-portal/app/election-breakdown/page.tsx

```diff
@@ -5,25 +5,62 @@ import Head from "next/head";
 import Image from "next/image";
 
 import styles from "./page.module.css";
-import Rightbar from "./modules/rightbar";
 import Banner from "./modules/banner";
 import Canvas from "./modules/canvas";
+import { redirect } from "next/navigation";
+import { Year } from "@/types/Year";
+import { RaceType } from "@/types/RaceType";
+import Rightbar from "@/components/rightbar";
+import { useRouter } from "next/navigation";
 
 export default function Election_Breakdown_Page() {
   const [isBannerVisible, setIsBannerVisible] = useState<boolean>(true);
-  const [drawMode, setDrawMode] = useState<boolean>(false);
+  const [raceType, setRaceType] = useState<RaceType>(RaceType.Presidential);
+  const [year, setYear] = useState<Year>(Year.TwentyTwo);
+  const [showRightbar, setShowRightbar] = useState<boolean>(true);
+  const [drawMode, setDrawMode] = useState<boolean>(true);
```

Comment:
I see the variables and toggle for draw mode, which is great, but it is not working on my end. My guess is maybe some canvas tag that isn't wrapped?

Reply:
Something like this on main branch: {drawMode && <Canvas />}. I would look to Louis's code